### PR TITLE
Fix copy of php modules

### PR DESCRIPTION
**Pull request by mannytoledo**
- modules were being overwritten during a copy
- Added goss test to check for the new relic key string replacement

### CODE REVIEW

File: container/root/tests/php-fpm/legacy.goss.yaml [hunk 1]
```diff
@@ -53,3 +53,9 @@ package:
     installed: true
   php-yaml:
     installed: true
+
+file:
+  /etc/php/5.6/mods-available/newrelic.ini:
+    exists: true
+    contains:
+      - "${REPLACE_NEWRELIC_LICENSE}"
```
Comment (mannytoledo) [inline container/root/tests/php-fpm/legacy.goss.yaml:62]: Really basic check for now but should always be true during a build. The file path would only change at runtime.

**Comment by bryanlatten** [comment]
👍

**Comment by andyroo2000** [comment]
👍

**Comment by ko-be** [comment]
👍

**Comment by bossjones** [comment]
👍

**Pull request merged**